lib/Service/OAuthService.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Stripe\Service;
+
+class OAuthService extends \Stripe\Service\AbstractService
+{
+    /**
+     * Sends a request to Stripe's Connect API.
+     *
+     * @param string $method the HTTP method
+     * @param string $path the path of the request
+     * @param array $params the parameters of the request
+     * @param array|\Stripe\Util\RequestOptions $opts the special modifiers of the request
+     *
+     * @return \Stripe\StripeObject the object returned by Stripe's Connect API
+     */
+    protected function requestConnect($method, $path, $params, $opts)
+    {
+        $opts = \Stripe\Util\RequestOptions::parse($opts);
+        if (!isset($opts->apiBase)) {
+            $opts->apiBase = $this->client->getConnectBase();
+        }
+
+        return $this->request($method, $path, $params, $opts);
+    }
+
+    /**
+     * Generates a URL to Stripe's OAuth form.
+     *
+     * @param null|array $params
+     * @param null|array $opts
+     *
+     * @return string the URL to Stripe's OAuth form
+     */
+    public function authorizeUrl($params = null, $opts = null)
+    {
+        $params = $params ?: [];
+
+        $base = ($opts && \array_key_exists('connect_base', $opts)) ? $opts['connect_base'] : $this->client->getConnectBase();
```

Comment:
I think we should use `api_base` instead of `connect_base` here. It would change the API from the existing `\Stripe\OAuth::authorizeUrl` static method, but it would make it consistent with the other two methods in the service. wdyt?

Reply:
Agreed.